src/backend/InvenTree/order/models.py

```diff
@@ -666,6 +666,16 @@ def company(self):
         help_text=_('Destination for received items'),
     )
 
+    delivery_address = models.ForeignKey(
+        Address,
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+        verbose_name=_('Delivery Address'),
+        help_text=_('Delivery address for this order'),
+        related_name='+',
```

Comment:
This is not practical for future use; please use a related name that makes sense when using the Adress model